packages/opal-common/opal_common/config.py

```diff
@@ -208,6 +208,12 @@ class OpalCommonConfig(Confi):
         description="The client to use for fetching data, can be either aiohttp or httpx."
         "if provided different value, aiohttp will be used.",
     )
+    HTTPX_FETCHER_TIMEOUT = confi.int(
+        "HTTPX_FETCHER_TIMEOUT",
+        5,
+        description="The timeout for the httpx fetcher provider, in seconds. "
+        "if provided different value, 5 seconds will be used.",
+    )
```

Comment:
I think it is better to make the timeout "fetcher agnostic" - it doesn't really matter which fetcher client you are using, if you want to have timeout then it's supposed to be the same if you use httpx or aiohttp.
Imagine you use HTTPX client then you switch back to aiohttp for some reason, the timeout magically disappears and you must remember you need to configure some other env var ( or it might even not be possible at all )

Reply:
would name it HTTP_FETCHER_TIMEOUT 😃